packages/typespec-test/test/loadTest/spec/models.tsp

```diff
@@ -836,13 +836,11 @@ model DimensionValue {
 @parentResource(Dimension)
 model DimensionValueList {
   @key
-  value: {
-    items: DimensionValue[]
-  };
```

Comment:
is this change come from service team ?

Reply:
The change is not from me, yes, this mainly corrects the wrong definition.